p2p/discover/node.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/ethereum/go-ethereum/crypto/secp256k1"
 )
 
-const nodeIDBits = 512
+const NodeIDBits = 512
```

Comment:
Hmm, I figured that AddressLength and HashLength are already exposed, so why not expose this too. Any particular reason against is?